app/models/project.rb

```diff
@@ -221,7 +221,7 @@ def set_last_synced_at
   def async_sync
     return unless platform_class_exists?
 
-    sync_classes.each { |sync_class| PackageManagerDownloadWorker.perform_async(sync_class.name, name, nil, "project") }
+    sync_classes.each { |sync_class| PackageManagerDownloadWorker.perform_async(sync_class.name, name, nil, "project", 0, true) }
```

Comment:
We are limited because this is a sidekiq worker and adding this option to the method signature requires I set that `0` value. I've updated it to take the flag in from the manual_sync method. I must have forgot to update that after testing locally.